torchci/scripts/no_flaky_tests_alert.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+import json
+import os
+from datetime import datetime, timedelta
+from typing import Any, Dict, List, Optional
+
+import requests
+from check_alerts import (
+    GRAPHQL_URL,
+    HEADERS,
+    PYTORCH_REPO_NAME,
+    REPO_OWNER,
+    create_issue,
+)
+
+FLAKY_TESTS_LABEL = "module: flaky-tests"
+NO_FLAKY_TESTS_LABEL = "no-flaky-tests-alert"
+FLAKY_TESTS_SEARCH_PERIOD_DAYS = 7
+
+NUM_ISSUES_QUERY = """
+query ($query: String!) {
+  search(type: ISSUE, query: $query) {
+    issueCount
+  }
+}
+"""
+
+def get_num_issues_with_label(owner: str, repo: str, label: str, from_date: Optional[str]= None) -> int:
+    query = f'repo:{owner}/{repo} label:"{label}" is:issue is:open'
```

Comment:
I think we might need to run this less frequently than other alert (regular, queue), mainly because search query to GitHub is expensive and can be rate limited.  We also don't need to check if there is any new flaky test every 5 minutes.  IMO, this can be run once per day

Reply:
i got rid of the flaky stuff here because it was a lot of changes, I'll make another PR for various changes to the no flaky test alert later